connectivity/netsocket/tests/UNITTESTS/netsocket/DTLSSocket/CMakeLists.txt

```diff
@@ -12,19 +12,12 @@ target_compile_definitions(${TEST_NAME}
 
 target_sources(${TEST_NAME}
     PRIVATE
-        ${mbed-os_SOURCE_DIR}/connectivity/netsocket/source/SocketAddress.cpp
```

Comment:
In the commit "Unittests: Clean up unused sources from netsocket" how did we determine the files weren't used? Please add the reasoning to the commit message. 

It also looks like we added linking to `mbed-stubs-nanostack-libservice`. Did we move the "unused sources" to that library in some other commit/PR, or are they actually unused? I can't tell from the commit description.

Reply:
The files were determined to be unused by removing them from `target_sources` and verifying that the test still compiled and ran successfully. I understood from speaking to Rajkumar that this was sufficient reasoning, I will add the reasoning to the commit message description. 

The linking to `mbed-stubs-nanostack-libservice` was to use the stubs of `stoip4.c` and `ip4tos.c` instead of including those source files. I can also amend the commit message to explain this, or would it be more appropriate to move these changes to a separate commit ?